test/realm/test-import-expression.js

```diff
@@ -52,7 +52,13 @@ const whitespace = `const a = import ('evil')`;
 
 const comment = `const a = import/*hah*/('evil')`;
 
-const doubleslashcomment = `const a = import // hah
+const doubleSlashComment = `const a = import // hah
+('evil')`;
+
+const htmlOpenComment = `const a = import ${'<'}!-- hah
```

Comment:
Rejection of HTML comments should also be tested without import expressions.
Rejection of direct eval needs to be tested.

Reply:
Done, in next commit.